rest/route/stats.go

```diff
@@ -514,12 +519,12 @@ func (m *cedarTestStatsMiddleware) ServeHTTP(rw http.ResponseWriter, r *http.Req
 	resp, err := c.Do(req)
 	if err != nil {
 		gimlet.WriteResponse(rw, gimlet.MakeJSONInternalErrorResponder(errors.Wrap(err, "problem sending test stats request to cedar")))
+		return
 	}
 	defer func() {
 		_ = resp.Body.Close()
 	}()
 	if resp.StatusCode == http.StatusOK {
-		rw.WriteHeader(http.StatusOK)
 		for key, vals := range resp.Header {
 			for _, val := range vals {
 				rw.Header().Add(key, val)
```

Comment:
Does this already set the content-type to application/json? My only concern is that it `io.Copy` would write the data without specifying that it's returning JSON.

Reply:
Yes, because cedar sets it as JSON. So we would be copying over that header.